query/redis.sp

```diff
@@ -0,0 +1,39 @@
+query "redis_instance_auth_enabled" {
+  sql = <<-EOQ
+    select
+      type || ' ' || name as resource,
+      case
+        when (arguments ->> 'auth_enabled') = 'true' then 'ok'
+        else 'alarm'
+      end as status,
+      name || case
+        when (arguments ->> 'auth_enabled') = 'true' then ' AUTH enabled'
+        else ' AUTH disabled'
```

Comment:
```suggestion
        when (arguments ->> 'auth_enabled') = 'true' then ' Auth enabled'
        else ' Auth disabled'
```

Reply:
@Priyanka-Chatterjee-2000 @madhushreeray30 what kind of Auth is being checked over here?